public/ledger.js

```diff
@@ -350,6 +361,85 @@ class Ledger {
       await transport.send(ledgerCLA, this.commands.SEND_TX, 2, 0);
     }
   }
+
+  /**
+   * Get signature for token on ledger
+   *
+   * @param {Object} data Data to send to Ledger
+   *   version + uid + len(symbol) + symbol + len(name) + name
+   *
+   * @return {Promise} Promise resolved when signature is received
+   */
+  signToken = async (data) => {
```

Comment:
Should we change to `getTokenSignature`? I think it's more explicit what the method does, just like you wrote in the docstring. If we change this, then we should change the command as well

Reply:
Maybe, but I think it's good to maintain consistency with the method name and the command name

Reply:
I agree. And to change the command name we must change the ledger app code?

Reply:
Yes.